src/hooks/useAllRolesData/useAllRolesData.js

```diff
@@ -20,7 +20,7 @@ function useAllRolesData() {
   const [namespace] = useNamespace();
 
   const { data, isLoading, isSuccess } = useQuery([namespace, 'user-roles'], () => {
-    return ky.get(`roles?limit=${stripes.config.maxUnpagedResourceCount}&query=cql.allRecords=1 sortby name`).json();
+    return stripes.hasInterface('roles') && ky.get(`roles?limit=${stripes.config.maxUnpagedResourceCount}&query=cql.allRecords=1 sortby name`).json();
```

Comment:
This works fine. A more conventional approach is to set the `enabled` attribute of the object passed to `useQuery`, e.g. `useQuery({ ..., enabled: stripes.hasInterface('roles')})`.